### PR TITLE
[Bug] scrollable legends #146

### PR DESCRIPTION
**Pull request by nxanil**
@natabal 

Fix issue https://github.com/nuagenetworks/vis-graphs/issues/146 for barGraph, LineGrpah, AreaGraph.Please have a look.

### CODE REVIEW

**Comment by natabal** [comment]
@nxanil I tested this bug against some of our testbeds here and we need as vertical and horizontal scrolls. See attached.
Screen Shot 2019-04-22 at 12 27 57 PM

**Comment by nxanil** [comment]
@natabal 
To enable scrolling on legends data, we have to add `separate` property in legend configuration.
For more info here is docs  - 
[Screenshot from Gyazo](https://gyazo.com/3936750d69103657addfaa4310af9b45)

**Comment by natabal** [comment]
@nxanil I think it's better to make separate as default (20) as it does not require vis-config changes for every single graph! Unfortunately there are many graphs that might require this.

**Comment by nxanil** [comment]
@natabal 
Made `separate:20` as default property of legend and update the docs. Please have a look.

File: Graphs/PieGraph/default.config.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ export const properties = {
     pieInnerRadius: 0,      // The inner radius of the slices. Make this non-zero for a Donut Chart.
     pieOuterRadius: 0.8,    // The outer radius of the slices.
     pieLabelRadius: 0.85,   // The radius for positioning labels.
+    legend: {
```
Comment (natabal) [inline Graphs/PieGraph/default.config.js]: We cannot make this false by default!

**Comment by nxanil** [comment]
@natabal 
All the feedback changes have been done. Please have a look.

**Comment by natabal** [comment]
@nxanil As mentioned please keep the position on the visualization for the legends! Now it's on the left side instead of the right side! Please provide a fix asap! Thanks

**Comment by nxanil** [comment]
@natabal 
Legend position issue has been fixed now. Please have a look.

**Comment by natabal** [comment]
@nxanil It seems that the width is now fixed no matter how much I give to 'separate'

**Comment by nxanil** [comment]
@natabal Lets connect when you will be around.

**Pull request closed without merging**